packages/eslint-plugin/tests/rules/ban-types.test.ts

```diff
@@ -23,6 +23,25 @@ const options: InferOptionsTypeFromRule<typeof rule> = [
     },
   },
 ];
+                                        
+const options2: InferOptionsTypeFromRule<typeof rule> = [
+  {
+    types: {
+      'null': {
+        message: 'Use undefined instead.',
+        fixWith: 'undefined',
```

Comment:
Are fixers expected to be safe to run on a code base?  Simply substituting `undefined` for `null` is likely to break a lot of code.

Reply:
the fixer for this rule is optional, user defined, and separated per type.

The intention for these fixers is not that you'd turn on the rule with fixers on a 10k line codebase and hope for the best, it's that you'd do it progressively via providing no fixer first:
```
'null': {
  message: 'Use undefined instead.'
}
// or
'null': 'Use undefined instead.'
```
Then adding in the fixer when you're ready.